server/container/cli/src/main/java/org/apache/james/cli/ServerCmd.java

```diff
@@ -102,7 +105,7 @@ public static void main(String[] args) {
         } catch (IOException ioe) {
             failWithMessage("Error connecting to remote JMX agent : " + ioe.getMessage());
         } catch (Exception e) {
-            e.printStackTrace();
+            LOG.error("Error on command: ", e);
             failWithMessage("Error while executing command:" + e.getMessage());
```

Comment:
As I traced on that, the message is not null. The exception class is NullPointerException but on another part. So think the exception class is not enough. I changed a bit to print out stasktrace in the log file.